invoicer_project/user/userserializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from .models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    repeat_password = serializers.CharField(max_length=100, write_only=True)
+
+    class Meta:
+        model = User
+        fields = ("first_name", "last_name", "email", "password", "repeat_password")
+        extra_kwargs = {'password': {'write_only': True}}
+
+    def validate(self, attrs):
+        password = attrs.get('password')
+        password2 = attrs.get('repeat_password')
```

Comment:
Let's think of the name instead of password2

Reply:
Well, I can do it, but I don't see the point in this